packages/bot-web-ui/src/assets/sass/summary-card.scss

```diff
@@ -0,0 +1,44 @@
+.db-contract-card {
```

Comment:
Do we still need this? It doesn't come with the `<ContractCard>` component itself?

Reply:
I've renamed it to `db-summary-card`, which makes more sense now.